setup.py

```diff
@@ -61,8 +61,11 @@
     'elasticsearch>=1.3.0',
     'fixture>=1.5',
     'intbitset>=2.2.1',
+    'invenio-access>=0.1.0',
```

Comment:
we should not add `invenio-*` as required packages here. All signal handlers should be moved to appropriate modules.